README.md

```diff
@@ -21,21 +21,25 @@ docker run --env-file .env --rm -p 8080:8080 srcd/code-annotation
 
 ### Non-docker
```

Comment:
You should put a standard path to place the binary. 

Something like: 
```
wget https://github.com/mcuadros/passage/releases/download/v0.1.0/passage_v0.1.0_linux_amd64.tar.gz
tar -xvzf passage_v0.1.0_linux_amd64.tar.gz
cp passage_v0.1.0_linux_amd64/passage /usr/local/bin/
```

Reply:
agreed. But I want to postpone it until we have our first release.